### ISSUE TITLE
Component Request: uv-e-relationalFromSql and uv-t-relationalToRdf

### ISSUE THREAD

**Issue by skarampatakis** [linkedpipes/etl#184]
It would be nice if we could have these components on LP. Are there any thoughts on this and an actual timeplan for implementation? 

Thank you in advance.

**Comment by jakubklimek** [comment]
As mentioned in the other issue, there is no plan for supporting for relational data in the pipeline itself, i.e. uv-e-relationalFromSQL produces relational data and uv-t-relationalToRdf transforms it to RDF. However, the functionality of these two components (together) can be implemented in LP-ETL in a clean way as extractors from relational database directly to RDF utilizing the [Direct Mapping](https://www.w3.org/TR/rdb-direct-mapping/) and [R2RML](https://www.w3.org/TR/r2rml/) standards.

The implementation involves selecting an existing implementation (such as [db2triples](https://github.com/antidot/db2triples)) suitable for wrapping in an LP-ETL component and the actual wrapping.

Regarding the timeline, is there any specific deadline until which you need it? Or is it fine if it is by the end of July?

**Comment by skarampatakis** [comment]
I am not aware of db2triples, but I think it could be OK if it has the same functionality and behaviour, or even better, as the mentioned DPUs  in UV.

As for the time, no I don't have any deadline, I am currenlty doing my work using UV. Although it would be better if I could stick in one tool :)

**Comment by jakubklimek** [comment]
OK, could you describe your use case a bit? Do you use these two DPUs solely together and then process the resulting RDF data using SPARQL?

**Comment by skarampatakis** [comment]
Yes, that's the case. I beleive a 'quick and dirty' solution would be to create somehow a csv dump of sql query results and then load it to tabular.

**Comment by jakubklimek** [comment]
Exactly. But given that we have some time, I would do it properly.

**Comment by skarampatakis** [comment]
Take your time! No need to hurry at all.
